src/commcare_cloud/fab/operations/formplayer.py

```diff
@@ -132,3 +132,11 @@ def _get_old_formplayer_builds(build_dir):
         old_builds = sorted(_get_builds(previous_build_paths.split('\n')), reverse=True)
         old_builds.remove(current_build)
         return old_builds
+
+
+def formplayer_is_running_from_old_release_location():
+    ps_formplayer = sudo('ps aux | grep formplaye[r]')
+    if env.code_current in ps_formplayer:
```

Comment:
nit
```suggestion
    return env.code_current in ps_formplayer
```
and drop remaining lines in function

Reply:
https://github.com/dimagi/commcare-cloud/pull/3279